### PR TITLE
fix(authz-keycloak): set permissions as empty table when lazy_load_path is false

### PR DESCRIPTION
**Pull request by tzssangglass**
### What this PR does / why we need it:

fix: #4840 
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/plugins/authz-keycloak.lua [hunk 1]
```diff
@@ -83,6 +84,9 @@ local schema = {
                 {
                     properties = {
                         lazy_load_paths = {enum = {false}},
+                    },
+                    anyOf = {
+                        {required = {"permissions"}}
```
Comment (spacewander) [inline apisix/plugins/authz-keycloak.lua:89]: We don't need to require it as we already provide a default value

Comment (tzssangglass) [reply apisix/plugins/authz-keycloak.lua]: got

File: t/plugin/authz-keycloak.t [hunk 2]
```diff
@@ -152,7 +152,33 @@ done
 
 
 
-=== TEST 6: full schema check
+=== TEST 6: no permissions when lazy_load_paths=false
```
[the remark this answers is not in the document]
Comment (tzssangglass) [reply t/plugin/authz-keycloak.t]: I deleted the test case ...... I can't pull up the keyclock container process locally ......

[the remark this answers is not in the document]
Comment (spacewander) [reply t/plugin/authz-keycloak.t]: If I am correct, the previous code will be broken at https://github.com/apache/apisix/blob/b72404f643f37b10eb164056d784c33a97251c46/apisix/plugins/authz-keycloak.lua#L591 even without reaching keyclock.

[the remark this answers is not in the document]
Comment (tzssangglass) [reply t/plugin/authz-keycloak.t]: got

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**